addons/intelitems/functions/fnc_handleLoadout.sqf

```diff
@@ -39,7 +38,7 @@ private _magazines = magazines _player;
             };
         } forEach _magazineIds;
     };
-} forEach (_magazines arrayIntersect _magazines);
+} forEach ([_player, 2] call EFUNC(common,uniqueItems));
```

Comment:
Cache is invalidated in `common` upon loadout change, so it should be fine to use this.